lambda-buffers-frontend/build.nix

```diff
@@ -90,13 +90,13 @@
         lbf = pkgs.writeShellScriptBin "lbf" ''
           export LB_CODEGEN=${config.packages.lbg-haskell}/bin/lbg-haskell;
           export LB_COMPILER=${config.packages.lbc}/bin/lbc;
-          ${config.packages.lbf-pure}/bin/lbf $@
+          ${config.packages.lbf-pure}/bin/lbf "$@"
         '';
```

Comment:
Don't we want to have multiple arguments here?

I call `lbf` with a bunch of .lbf files; `lbf Foo.lbf Bar.lbf Baz.lbf`

I think this would make all these into a single argument rather than 3?

Reply:
Woohooo today I learned!